homework-09/main.js

```diff
@@ -0,0 +1,154 @@
+// 1
+function Human(name, age, gender, nationality, country, listOfCountries) {
+  this.name = name;
+  this.age = age;
+  this.gender = gender;
+  this.nationality = nationality;
+  this.country = country;
+  this.listOfCountries = listOfCountries;
+};
+
+function greeting(greet) {
+  greet = `Hello! My name is ${this.name} I'm ${this.age}, ${this.gender}
+              I'm from ${this.country}. My nationality is ${this.nationality}
+              I'd like to visit ${this.listOfCountries} countries`;
+  return greet;
+};
+
+function wakeUp(awake) {
+  awake = `Good morning! Have a good day ${this.name}`;

```

Comment:
And the same here - the general idea is that functions accepts no arguments but it will dynamically determine what is the value of this based on apply/call call

```suggestion
function wakeUp() {
  const awake = `Good morning! Have a good day ${this.name}`;

```